sycl/test/check_device_code/const_ref_image_accessor.cpp

```diff
@@ -0,0 +1,32 @@
+// RUN: %clangxx -I %sycl_include -S -emit-llvm -fsycl -fsycl-device-only %s -o - | FileCheck %s
```

Comment:
```suggestion
// RUN: %clangxx -S -fsycl -fsycl-device-only %s -o - | FileCheck %s
```
`-I` and `-emit-llvm` are not necessary when `-fsycl` and `-fsycl-device-only` are set.

What is `FileCheck` is called for?  I don't see any checks inside the code.

Reply:
Errors in commented section for testing purposes were corrected in new commit.

Reply:
Okay. I mark this as resolved, but I created another to comment to fix errors added in new commit.